src/meta/src/barrier/info.rs

```diff
@@ -22,16 +22,9 @@ use crate::barrier::Command;
 use crate::manager::{ActiveStreamingWorkerNodes, ActorInfos, InflightFragmentInfo, WorkerId};
 use crate::model::{ActorId, FragmentId};
 
-#[derive(Debug, Clone)]
-pub(crate) struct CommandNewFragmentInfo {
-    pub new_actors: HashMap<ActorId, WorkerId>,
-    pub table_ids: HashSet<TableId>,
-    pub is_injectable: bool,
-}
-
 #[derive(Debug, Clone)]
 pub(crate) enum CommandFragmentChanges {
-    NewFragment(CommandNewFragmentInfo),
+    NewFragment(InflightFragmentInfo),
```

Comment:
Why remove `CommandNewFragmentInfo`?

Reply:
The `CommandNewFragmentInfo` is exactly the same as `InflightFragmentInfo`, so using `InflightFragmentInfo` is enough.